### DISCUSSION TITLE
Cooking with Monika - Recipe Suggestions

### DISCUSSION THREAD

**Discussion by Babushka93** [Monika-After-Story/MonikaModDev#7916] [general]
#2164 talks about cooking with Monika,

However I'm wondering two things, firstly if Monika gave you cooking instructions would she only give you vegetarians options since she herself is vegetarian and has previously recommended the player to try it.

Secondly, what recipes do you think she would give to the player, feel free to write some below.

I'm thinking of having a go of doing a cooking menu similiar to the story menu, where new recipes can be unlocked but I need ideas for recipes

**Comment by Babushka93** [comment]
Ideas so far:

Stir Fry Vegetables
Omurice (Omelette Rice)
Oyakodon (Chicken & Egg Bowl)
Rocky Road Parfaits

**Comment by mosquitoe** [comment]
What about fried squid?
Wait, she's a vegetarian

**Comment by AgentZephyr** [comment]
Miso soup
Salad Stuff
I like the idea!

**Comment by Reviani** [comment]
Curry
Potato Stew (and really, any potato dish, potatos are good)
Stuffed Mushroom
veggie stew

And there's many tpes of pastas but hard to say if monnika would mention them much as they can be a bit unhealthy. But all in all this seems like a fun idea, if possible

**Comment by tabtlenta** [comment]
I love it. I think it would make sense for this to be accessible by a "What's your favourite food?" dialogue option, and for her to say something like "it's so hard to pick a favourite, but I used to make this dish..." and then walking you through a randomly-selected recipe (kinda like the "Fun facts" topic).

She also mentions loving the free samples at chocolate stores, so maybe she has a simple chocolate recipe to share with us? And I'm sure she knows how to brew the perfect cup of coffee!

**Comment by tabtlenta** [comment]
Oh, and maybe she asks you each time you select the recipe topic "Did you make/like my last recipe?" - wherein if you say you didn't like it she no longer repeats it, or if you say you made it and liked it you get a tiny affection boost

**Comment by ThePotatoGuy** [comment]
[keeping discussion open as suggestions for the cooking monika issue]
